lib/ansiblelint/utils.py

```diff
@@ -784,7 +784,7 @@ def get_yaml_files(options):
             stderr=subprocess.STDOUT,
             universal_newlines=True
         ).split()
-    except subprocess.CalledProcessError as exc:
+    except (subprocess.CalledProcessError, FileNotFoundError) as exc:
```

Comment:
I think you need to also customize access to `exc.output` vs. `str(exc)` because `FileNotFoundError` probably doesn't have `.output` attr.

Reply:
Sorry. Didn't check verbosity option. Fixed.